assets/dev/js/editor/elements/views/container.js

```diff
@@ -138,9 +138,10 @@ const ContainerView = BaseElementView.extend( {
 				elementor.getPreviewView().onPanelElementDragEnd();
 
 				const draggedView = elementor.channels.editor.request( 'element:dragged' ),
-					draggingInSameParent = ( draggedView?.parent === this );
+					draggingInSameParent = ( draggedView?.parent === this ),
+					hasInnerContainer = jQuery( event.currentTarget ).hasClass( 'e-container__inner' );
 
-				let $elements = jQuery( event.currentTarget ).hasClass( 'e-container__inner' ) ? jQuery( event.currentTarget.parentElement.parentElement ).find( '> .elementor-element' ) : jQuery( event.currentTarget.parentElement ).find( '> .elementor-element' );
+				let $elements = hasInnerContainer ? jQuery( event.currentTarget.parentElement.parentElement ).find( '> .elementor-element' ) : jQuery( event.currentTarget.parentElement ).find( '> .elementor-element' );
```

Comment:
I also said this in the previous comment - please add another variable for the container selector and use that like this:
```
const containerSelector = hasInnerContainer ? event.currentTarget.parentElement.parentElement : event.currentTarget.parentElement;

let $elements = jQuery( containerSelector ).find( '> .elementor-element' )
```

Reply:
updated